src/main/java/com/jherrick/BasicBoardEvaluator.java

```diff
@@ -1,28 +1,90 @@
 package com.jherrick;
 
 public class BasicBoardEvaluator implements BoardEvaluator{
+    
+    // =============== Instance variables ===============
+    
+    // Evaluation metrics. Only add or remove metrics, do not hard code in. Should be hard coded in EVAL_CONSTANTS
+    private static int c0 = Constants.EVAL_CONSTANTS[0];
+	private static int c1 = Constants.EVAL_CONSTANTS[1];
+	// private static int c2 = Constants.EVAL_CONSTANTS[2];
+
+    public BigBoard board;
+    public Side side;
+
+
+
+    // ============== Constructors ======================
```

Comment:
The spacing on some of these separators is a little funky. I tend to do 10 = on each side instead of making them all line up on the right side. 

Also, I tend only to add them for the public-facing methods and the helpers. Instance variables and constructors feel pretty self-explanatory when I'm reading code. But that's personal taste - if having all the separators is helpful for you, I'm happy to leave them in.